content/en/docs/tasks/federation/administer-federation/deployment.md

```diff
@@ -73,7 +73,7 @@ if you have 3 registered clusters and you create a Federated Deployment with
 `spec.replicas = 9`, then each Deployment in the 3 clusters will have
 `spec.replicas=3`.
 To modify the number of replicas in each cluster, you can specify
-[FederatedReplicaSetPreference](https://github.com/kubernetes/federation/blob/{{< param "githubbranch" >}}/apis/federation/types.go)
+[FederatedReplicaSetPreference](https://github.com/kubernetes-retired/federation/blob/master/apis/federation/types.go{{< param "githubbranch" >}})
```

Comment:
@Aresforchina I just tried this link in the preview. Can you recheck if this works? Thanks.

Reply:
@kbhawkey Of course, I've already rechecked it

Reply:
Hello @Aresforchina . I retried the link again. This link worked for me, _not_ the link above:

https://github.com/kubernetes-retired/federation/blob/master/apis/federation/types.go

This code is retired. In the short term, the link should be fixed on the page.
I logged an issue (some time ago) to update the federation pages to the new federation information (https://github.com/kubernetes-sigs/kubefed).

Reply:
Did you remove the federation change from this PR?